lib/semantic_range.rb

```diff
@@ -221,6 +224,16 @@ def self.lte?(a, b, loose: false)
     compare(a, b, loose: loose) <= 0
   end
 
+  def self.valid?(version, loose: false)
+    return false unless version.is_a?(String)
+
+    stripped_version = version.strip
+    return false if stripped_version.length > MAX_LENGTH
+
+    rxp = loose ? LOOSE : FULL
+    return !!rxp.match(stripped_version)
```

Comment:
```suggestion
    return rxp.match?(stripped_version)
```
I think you can use the boolean method here instead of using `!!` to convert it.